Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM continuumio/anaconda3:latest
 
 ADD patches/ /tmp/patches/
+ADD nbconvert-extensions.tpl /tmp/nbconvert-extensions.tpl
```

Comment:
While it should work for the lifetime of a docker container, `/tmp/` is typically used for temporary files (ok do be deleted on reboot etc...).  Unless you had some reason of putting it here, how about using `/opt/kaggle/` or so?